scripts/prepare-release.js

```diff
@@ -64,9 +68,9 @@ const inputReleaseTag = async () => {
             await gitCheckMain();
             await gitCheckClean();
             await gitPullLatestFromMain();
-            const { tag, notes } = await inputReleaseTag();
+            const { version, tag, notes } = await inputReleaseTag();
             await gitCreateReleaseBranch(tag);
-            await updateTerragruntHcl(tag);
+            await updateEnvironmentManifest(version, 'staging');
```

Comment:
Maybe your looking to do this later --- should we be pulling in "staging" as a cli arg here?